tests/test_nwm22_grid_loading.py

```diff
@@ -14,14 +18,16 @@
 @pytest.mark.filterwarnings("ignore::RuntimeWarning")
 def test_grid_loading():
 
-    component_paths = ["gcs://national-water-model/nwm.20201218/forcing_analysis_assim/nwm.t00z.analysis_assim.forcing.tm00.conus.nc"] # noqa
+    component_paths = [
+        "gcs://national-water-model/nwm.20201218/forcing_analysis_assim/nwm.t00z.analysis_assim.forcing.tm00.conus.nc" # noqa
+    ]
 
     json_paths = build_zarr_references(component_paths,
                                        TEST_DIR,
                                        False)
 
     json_file = Path(TEST_DIR,
```

Comment:
@samlamont not sure where to best leave this comment, but in general, I'd say it is probably better to not include test output in the repo.  Have tests write to a folder with a .gitignore that ignores everything.